Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -154,16 +161,37 @@ class MainTabBarViewController: UITabBarController {
         let ribbonVC = RibbonViewController(rootViewController: segmentVC)
         
         let tabRoot = HeaderViewController(rootViewController: ribbonVC)
-        return tabViewController(root: tabRoot, title: "Settings", image: UIImage(named: "tab-icon-settings")!, tag: 2)
+        return tabViewController(root: tabRoot, title: "Settings", image: UIImage(named: "tab-icon-settings")!, tag: 2, isSettingsTab: true)
     }
 
-    private func tabViewController(root: UIViewController, title: String, image: UIImage, tag: Int) -> UIViewController {
+    //TODO: Find a better way to update tab than storing a reference here
+    private var settingsTabItem : UITabBarItem? = nil
+
+    private func tabViewController(root: UIViewController, title: String, image: UIImage, tag: Int, isSettingsTab: Bool = false) -> UIViewController {
         let nav = UINavigationController(rootViewController: root)
         let tabItem = UITabBarItem(title: title, image: image, tag: tag)
+
+        //TODO: Can we not use this haclk to have a reference to the UITabBarItem?
+        if isSettingsTab {
```

Comment:
When we receive an NSNotification about an unread conversation we want to update the badge of the settings tab bar item. What is the best way to get a reference to it?  To be used in `showBadge()` later.